c7n/resources/s3.py

```diff
@@ -1343,17 +1343,13 @@ class SetBucketReplicationConfig(BucketActionBase):
 
     def process(self, buckets):
         with self.executor_factory(max_workers=3) as w:
-            futures = {}
-            results = []
-            for b in buckets:
-                futures[w.submit(self.process_bucket, b)] = b
-            for f in as_completed(futures):
-                if f.exception():
-                    b = futures[f]
-                    self.log.error('error modifying bucket:%s\n%s',
-                                   b['Name'], f.exception())
-                results += filter(None, [f.result()])
-            return results
```

Comment:
Hey @kapilt, good point. I hadn't considered the use case for the result values until now. I have removed the return